src/notification/notification.js

```diff
@@ -172,6 +180,23 @@ angular.module('patternfly.notification').provider('Notifications', function ()
       }
     };
 
+    notifications.remove = function (notification) {
+      notifications.removeIndex($rootScope.notifications.data.indexOf(notification));
+    };
+
+    notifications.removeIndex = function (index) {
+      $rootScope.notifications.remove(index);
+    };
+
+    notifications.setViewing = function (notification, viewing) {
+      notification.viewing = viewing;
+      if (!viewing && !notification.show) {
+        notifications.remove(notification);
+      }
+    };
+
+    notifications.data = $rootScope.notifications.data;
```

Comment:
@jeff-phillips-18 curious why we still need to use $rootScope to pass thru the data?  Can we not just reference `Notifications.notifications.data` in `pfNotificationList` directive? I guess I'm OK with exposing them in $rootScope, just wasn't quite sure based on the example.